runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -207,6 +211,23 @@ private void rebuildAsync(boolean updated, boolean skillPaused, XpSnapshotSingle
 				? "200M"
 				: "Lvl. " + xpSnapshotSingle.getEndLevel());
 
+			// Add intermediate level positions to progressBar
+			ArrayList<Double> positions = new ArrayList<>();
+			if (xpTrackerPlugin.getXpTrackerConfig().showIntermediateLevels() && xpSnapshotSingle.getEndLevel() - xpSnapshotSingle.getStartLevel() > 1)
```

Comment:
You can simply inject xpTrackerConfig, or send it via ctor here, remove the xpTrackerPlugin call